c/polyjuice.h

```diff
@@ -701,6 +701,51 @@ void emit_log(struct evmc_host_context* context, const evmc_address* address,
   return;
 }
 
+/**
+ * check address collision
+ * check existence of eth_addr
+ * - if exists, then check nonce > 0 or it's NOT an EOA: return AddressCollision error
```

Comment:
I think the comment is not clear enough.
Is that means:
`if it's an EoA address with non-zero nonce or it's an contract account address, it can't be overwrite?`

Reply:
Yes. That will be more clear.